image_editor/core/image_processing/Image_Processing.py

```diff
@@ -75,3 +79,38 @@ def __create_circular_mask(self, height, width, center=None, radius=None):
 
         mask = dist_from_center <= radius
         return mask
+
+    def scale_image(self, **kwargs) -> str:
+        """
+        This method up scales / down scales an image by dimensions provided
+        :param kwargs: arguments to scale the image to some dimension
+        :return: scaled image path
+        """
+        img = cv.imread(self.img_path, cv.IMREAD_COLOR)
+        scale_amount = abs(float(kwargs.get('amount', '1')))
+
+        if scale_amount >= 1:  # upscale
+            img = cv.resize(img, None, fx=scale_amount, fy=scale_amount, interpolation=cv.INTER_CUBIC)
+
+        elif scale_amount > 0:  # downscale
+            img = cv.resize(img, None, fx=scale_amount, fy=scale_amount, interpolation=cv.INTER_AREA)
+
+        return cv_to_image(image_cv=img, image_path=self.img_path)
+
+    def flip_image(self, **kwargs) -> str:
+        """
+        This method flips image horizontally / vertically based on dimensions provided
+        :param kwargs: arguments to flip the image to some dimension
+        :return: flipped image path
+        """
+        img = cv.imread(self.img_path, cv.IMREAD_COLOR)
+        flip_direction = kwargs.get('direction', ie.FLIP_HORIZONTAL)
+
+        if flip_direction == ie.FLIP_VERTICAL:
+            img = cv.flip(img, 0)
+        elif flip_direction == ie.FLIP_HORIZONTAL:
+            img = cv.flip(img, 1)
+        elif flip_direction == ie.FLIP_VERTICAL_HORIZONTAL:
+            img = cv.flip(img, -1)
```

Comment:
Can the enum value be made as a third param to `cv.flip`
```suggestion
            img = cv.flip(img,  flip_direction)
```